api/src/main/resources/application-template.conf

```diff
@@ -0,0 +1,103 @@
+archway.featureFlags = ""
```

Comment:
@martin-morek can you leave in the defaults where they make sense? Just remove our cluster-specific configuration like hostnames and passwords

Reply:
Yes, sure!